querybook/server/tasks/all_tasks.py

```diff
@@ -54,3 +59,12 @@ def configure_workers(sender=None, conf=None, **kwargs):
         clean_up_query_execution()
     else:
         LOG.info(f"Starting DEV Celery worker: {sender}")
+
+
+@task_failure.connect
+def handle_task_failure(sender, signal, *args, **kwargs):
+    task_name = sender.name
+    if task_name.startswith("tasks.run_datadoc"):
```

Comment:
there is a task_type field for task and it is either "prod" or "user". Can you make the two connect?